docs/Developers/Newcomers_Guide.rst

```diff
@@ -23,6 +23,8 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
+Note: Please merge only *one* ``difficulty/newcomer`` Pull Request
```

Comment:
Actually, this note is intended to be written for contributors (they don't merge PRs :wink:)

Please, rephrase this to something like - 
"Please, don't take up more ``difficulty/newcomer`` issues once you have already solved one. It is recommended to move to higher difficulty issues after that."
:smile:

Reply:
I used the word 'merge' because it has been used even in ```merge a ``difficulty/newcomer`` Pull Request```.
I'll change my line, please tell me whether I should change the instruction list too.

Reply:
Ahh... I am not sure for that then :sweat_smile:
But anyway, your changes are perfect :smile: :+1: